language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/List.hs

```diff
@@ -174,3 +177,32 @@ getBuiltinSum s = do
         $ [ TyInst () (Constant () (BuiltinName () AddInteger)) $ TyInt () s
           , Constant () $ BuiltinInt () s 0
           ]
+
+-- |  'sumNat' as a PLC term.
+getBuiltinNatSum :: Natural -> Quote (Term TyName Name ())
```

Comment:
I don't think this should be in the standard library. That's not something that people would use for programming -- just for tests. I like it where it was.

Reply:
Fair enough - I thought these sort of functions would be of interest to many people working with PLC, but yeah, they can stay in the test libs.

Reply:
Also I do not have anything against functions moved to `StdLib.Data.Nat`. Those are fine, I just need to fix the TODOs there.

Reply:
I think `getBuiltinNatToInteger` is fine, since it's an actual PLC term, whereas `getBuiltinIntegerToNat` converts a Haskell integer so should be in `Meta`.

Reply:
The plan is to make `getBuiltinIntegerToNat` convert a PLC integer. But it can be in `Meta` for now.

Reply:
Right, and then we'll want `getIntegerToBuiltinInteger` in `Meta` :)